azerothcore/azerothcore-wotlk
# fix(DB/linked_respawn): Link Rajaxx's waves respawns to Rajaxx

## Changes Proposed:
-  So that they won't respawn after he's dead
-  

## Issues Addressed:

- Closes unopened issue.

## SOURCE:


## Tests Performed:

- Tested ingame
- 


## How to Test the Changes:


1. Kill Rajaxx and his adds
2. Wait 2 hours or modify the spawntimesecs of his adds to check if they respawn ```UPDATE `creature` SET `spawntimesecs` = 1 WHERE `id1` IN (15387, 15344);```

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
